npm/test/upload.test.ts

```diff
@@ -56,137 +56,44 @@ const mockUploadFailure: () => void = () => {
-  uploadFiles.upload()
-
-  const uploadForm: HTMLFormElement | null = document.querySelector(
-    "#file-upload-form"
-  )
-  const uploadDataForm: HTMLFormElement | null = document.querySelector(
-    "#upload-data-form"
-  )
-
-  if (uploadForm && uploadDataForm) {
-    await uploadForm.submit()
-    expect(mockRetrieveFiles).toBeCalledTimes(1)
-    expect(mockUploadFilesSuccess).toBeCalledTimes(1)
-  }
+  await uploadFiles.uploadFiles([dummyFile], "12345")
 
   expect(consoleErrorSpy).not.toHaveBeenCalled()
+  expect(mockGoToNextPage).toHaveBeenCalled()
 
-  mockRetrieveFiles.mockRestore()
-  mockUploadFilesSuccess.mockRestore()
+  mockGoToNextPage.mockRestore()
   consoleErrorSpy.mockRestore()
 })
 
 test("upload function console logs error when upload fails", async () => {
   mockUploadFailure()
-  setUpUploadHTML()
 
   const uploadFiles = setUpUploadFiles()
   const consoleErrorSpy = jest
     .spyOn(console, "error")
     .mockImplementation(() => {})
 
-  const mockRetrieveFiles = jest
-    .spyOn(uploadFiles, "retrieveFiles")
-    .mockImplementation(() => {
-      return [dummyFile]
-    })
-
-  const mockUploadFilesSuccess = jest
-    .spyOn(uploadFiles, "uploadFilesSuccess")
-    .mockImplementation(() => {})
-
-  const uploadForm: HTMLFormElement | null = document.querySelector(
-    "#file-upload-form"
-  )
-
-  const uploadDataForm: HTMLFormElement | null = document.querySelector(
-    "#upload-data-form"
-  )
-  uploadFiles.upload()
-  if (uploadForm && uploadDataForm) {
-    await uploadForm.submit()
-    expect(mockRetrieveFiles).toBeCalledTimes(1)
-    expect(mockUploadFilesSuccess).toBeCalledTimes(0)
-  }
-
-  expect(consoleErrorSpy).toHaveBeenCalled()
-
-  mockRetrieveFiles.mockRestore()
-  mockUploadFilesSuccess.mockRestore()
-  consoleErrorSpy.mockRestore()
-})
-
-test("upload function console logs error when no consignment id provided", async () => {
-  const consoleErrorSpy = jest
-    .spyOn(console, "error")
-    .mockImplementation(() => {})
-  const uploadFiles = setUpUploadFiles()
-
-  const mockRetrieveFiles = jest
-    .spyOn(uploadFiles, "retrieveFiles")
-    .mockImplementation(() => {
-      return [dummyFile]
-    })
-
-  const mockUploadFilesSuccess = jest
-    .spyOn(uploadFiles, "uploadFilesSuccess")
-    .mockImplementation(() => {})
-  document.body.innerHTML =
-    '<div class="govuk-file-upload">' +
-    '<form id="file-upload-form">' +
-    '<button class="govuk-button" type="submit" data-module="govuk-button" role="button" />' +
-    "</form>" +
-    "</div>"
-
-  const uploadForm: HTMLFormElement | null = document.querySelector(
-    "#file-upload-form"
-  )
-  uploadFiles.upload()
-  if (uploadForm) {
-    await uploadForm.submit()
-    expect(mockRetrieveFiles).toBeCalledTimes(0)
-    expect(mockUploadFilesSuccess).toBeCalledTimes(0)
-  }
+  await uploadFiles.uploadFiles([dummyFile], "12345")
 
   expect(consoleErrorSpy).toHaveBeenCalled()
+  expect(mockGoToNextPage).toBeCalledTimes(0)
```

Comment:
Probably change this to the following so its more explicit when read: `expect(mockGoToNextPage).not.toHaveBeenCalled()`

Reply:
Sure, I'll remove it. Thanks